etc/cron.d/repo

```diff
@@ -1,14 +1,9 @@
-#awstat
-*/15 * * * * root /usr/local/awstats/wwwroot/cgi-bin/awstats.pl -update -config=repo.opensciencegrid.org >> /usr/local/awstats/log.txt
-
 #update all .mash files, once daily
+@reboot    root /usr/bin/update_mashfiles.sh > /var/log/repo/update_mashfiles.log 2> /var/log/repo/update_mashfiles.err
 50 7 * * * root /usr/bin/update_mashfiles.sh > /var/log/repo/update_mashfiles.log 2> /var/log/repo/update_mashfiles.err
 
 #update all mash repos, every half-hour
 1-59/30 * * * * root /usr/bin/update_all_repos.sh > /var/log/repo/update_all_repos.log 2> /var/log/repo/update_all_repos.err
 
 #update mirror
 29-59/30 * * * *  root /usr/bin/update_mirror.py > /var/log/repo/update_mirror.log 2> /var/log/repo/update_mirror.err
-
-#service monitor
-*/15 * * * * root /opt/service-monitor/repo/test.sh
```

Comment:
As far as i am concerned, feel free to drop this service monitor test.sh script also if you are going to drop the cron for it.